docs/upgrading.md

```diff
@@ -4,6 +4,49 @@
 Breaking changes  typically (sometimes we don't realise they are breaking) have "!" in the commit message, as per
 the [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#summary).
 
+## Upgrading to v3.4
+
+### feat!: Add entrypoint lookup. Fixes #8344
+
+Affected if:
+
+* Using the Emissary executor.
+* Used the `args` field for any entry in `images`.
+
+This PR automatically looks up the command and entrypoint. This implementation for config look-up was incorrect (it
+allowed you to specify `args` but not `entrypoint`). `args` has been removed to correct the behaviour.
+
+If you are incorrectly configured, the workflow controller will error on start-up. 
```

Comment:
We should probably add this to FAQs.

Reply:
Maybe include an example that will come up in Google results.